python/lance/_lib.pyx

```diff
@@ -12,31 +12,38 @@ from pathlib import Path
 from pyarrow import Table
 
 from pyarrow._dataset cimport (
-    CDataset,
-    CFileWriteOptions,
-    CScanner,
-    Dataset,
-    FileFormat,
-    FileWriteOptions,
+CDataset,
```

Comment:
this indent is weird.

Reply:
fixed. not sure why pycharm is doing that